contracts/ligo/src/fixed_price_sale_market.mligo

```diff
@@ -89,16 +89,14 @@ let buy_token(sale, storage: sale_param * storage) : (operation list * storage)
   in
   let tx_money_op = transfer_money(sale.tokens.money_token_address, sale.tokens.money_token_token_id, sale.price, Tezos.sender, seller) in
   let tx_nft_op = transfer_nft(sale.tokens.token_for_sale_address, sale.tokens.token_for_sale_token_id, Tezos.self_address, Tezos.sender) in
-  let tx_nft_remove_operator_op = remove_operator_nft(sale.tokens.token_for_sale_address, sale.tokens.token_for_sale_token_id, Tezos.self_address, Tezos.sender) in
   let new_s = Big_map.remove sale storage in
-  (tx_money_op :: tx_nft_remove_operator_op :: tx_nft_op :: []), new_s
```

Comment:
also add/remove operator should happened before/after market contract is invoked.
I did not verify test or actual code that interacts with the market contract

Reply:
I'll add the test to this branch. I've only added tests for the market contract that sells NFTs in tez.